### ISSUE TITLE
IPR006544 interpro2go

### ISSUE THREAD

**Issue by sjm41** [geneontology/go-annotation#3600]
P-type ATPase, subfamily V (IPR006544) is currently associated with the GO MF "ATPase activity (GO:0016887)".

From the InterPro description and checking of the Drosophila proteins (at least), I think this can be changed to the more specific:
ATPase-coupled cation transmembrane transporter activity (GO:0019829)

What do you think?

@sarach06 
@EBI-Hsinyu

**Comment by sarach06** [comment]
Hi,
Yes, I agree that the term can be improved so I've changed it for the more specific one. Thanks for pointing this.

Best regards,
Sara

**Comment by sjm41** [comment]
Hi @sarach06,
Reviewing this again, I think the "GO:0140358 P-type transmembrane transporter activity" term on the entry can be improved to the child term "P-type ion transporter activity | GO:0015662".

Steven.

**Comment by sarach06** [comment]
Hi Steven,
I had a look and this entry also includes Endoplasmic reticulum transmembrane helix translocases (such as Q9HD20 from human and P39986 (SPF1) from yeast) which, according to [https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8053355/], they were thought to function as a manganese or calcium transporters, however, they specifically bind moderately hydrophobic transmembrane with short hydrophilic lumenal domains that misinsert into the endoplasmic reticulum (PMID:32973005, PMID:32353073). SO I think that adding the child term would be not accurate for these proteins I mention. Let me know if you agree.
Best regards,
Sara

**Comment by sjm41** [comment]
Ah, I see.
Does that mean you need to remove/edit the existing GO terms on this entry, or are these OK:

ATPase-coupled monoatomic cation transmembrane transporter activity (GO:0019829)
P-type transmembrane transporter activity (GO:0140358)

**Comment by sarach06** [comment]
Hi Steven,
Sorry for my slow reply. I went through this entry again and did a couple of changes. I added PTHR45630 to this entry and removed the Pfam signature to create the N-terminal domain, which has several papers describing the structure (IPR047819). Also, I created two specific family entries for P5A-type and P5B-type ATPases, whose InterPro ids are IPR047820 (cd07543 signature) and IPR047821 (cd07542 signature), respectively, and they are children of IPR006544. I added GO:0015662 and GO:0019829 to P5B-type ATPase entry, as I removed GO:0019829 from IPR006544. Let me know if you agree. All these changes will be visible in the next InterPro release (94.0).
Best regards,
Sara
